site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -272,7 +272,7 @@ In order to use `helm test` command described in this section you need to set `h
 
 Check the Agones installation by running the following command:
 ```bash
-helm test my-release --cleanup
```

Comment:
For curiosity, was the option `--cleanup` removed from the new helm version so we removed it here?

Reply:
Yes, the cleanup is now automatic so they go rid of the flag.